packages/flex-plugin-scripts/src/scripts/start.ts

```diff
@@ -10,11 +10,39 @@ import getConfiguration, { ConfigurationType, WebpackType } from '../config';
 import compiler from '../utils/compiler';
 
 import run, { exit } from '../utils/run';
-import { Plugin } from '../config/devServer/pluginServer';
-import { CLIFlexConfiguration } from './check-start';
+import pluginServer, { Plugin } from '../config/devServer/pluginServer';
 
 const termSignals: NodeJS.Signals[] = ['SIGTERM', 'SIGINT'];
 
+export interface UserInputPlugin {
+  name: string;
+  remote: boolean;
+  version?: string;
+}
+
+export const _parseUserInputPlugins = (...args: string[]): UserInputPlugin[] => {
+  const userInputPlugins: UserInputPlugin[] = [];
+  const config = readPluginsJson();
+  let nameIndex = args.indexOf('--name');
+
+  while (nameIndex !== -1 && nameIndex <= args.length) {
```

Comment:
This will break if I were to provide `--name foo --something-else --name another`

Reply:
Instead, try

```js
for (let i = 0; i <args.length - 1; i++) {
  if (args[I] === '--name') {
    // args[i+1] should be the plugin-name
  }
}
```